redi-connect-front/src/components/organisms/Footer.tsx

```diff
@@ -1,83 +1,101 @@
 import React from 'react'
-import { Columns, Container, Section } from 'react-bulma-components'
+import { Element, Columns, Container, Section } from 'react-bulma-components'
 import { useTranslation } from 'react-i18next'
-import Icons from '../atoms/MediaIcons'
+import { ReactComponent as RediSchool } from '../../assets/images/redi-school-logo.svg'
+import { ReactComponent as Deloitte } from '../../assets/images/deloitte.svg'
+import SocialMediaIcons from '../atoms/MediaIcons'
 import './Footer.scss'
 
 const RediFooter = () => {
   const year = new Date().getFullYear()
   const { t } = useTranslation()
 
+  const supportLinks = [
```

Comment:
Let's move the link also to the translation file and read them as objects.

Reply:
done